src/ArcCommander/APIs/AssayAPI.fs

```diff
@@ -691,327 +692,231 @@ module AssayAPI =
+                    midInitials
+                    (getFieldValueByName  "Email"                       personArgs)
+                    (getFieldValueByName  "Phone"                       personArgs)
+                    (getFieldValueByName  "Fax"                         personArgs)
+                    (getFieldValueByName  "Address"                     personArgs)
+                    (getFieldValueByName  "Affiliation"                 personArgs)
+                    (getFieldValueByName  "Roles"                       personArgs)
+                    (getFieldValueByName  "RolesTermAccessionNumber"    personArgs)
+                    (getFieldValueByName  "RolesTermSourceREF"          personArgs)
+                    comments
             
-            //let studyIdentifier = getFieldValueByName "StudyIdentifier" personArgs
-
-            //let investigationFilePath = IsaModelConfiguration.tryGetInvestigationFilePath arcConfiguration |> Option.get
+            let assayIdentifier = getFieldValueByName "AssayIdentifier" personArgs
             
-            //let investigation = Investigation.fromFile investigationFilePath
-
-            //// TO DO: Implementation für Assay-Datenstruktur
-
-            //match investigation.Assays with
-            //| Some studies -> 
-            //    match API.Study.tryGetByIdentifier studyIdentifier studies with
-            //    | Some study -> 
-            //        match study.Contacts with
-            //        | Some persons -> 
-            //            if API.Person.existsByFullName firstName midInitials lastName persons then               
-            //                if verbosity >= 1 then printfn "Person with the name %s %s %s already exists in the investigation file" firstName midInitials lastName
-            //                persons
-            //            else
-            //                API.Person.add persons person                           
-            //        | None -> 
-            //            [person]
-            //        |> API.Assay.setContacts study
-            //        |> fun s -> API.Assay.updateByIdentifier API.Update.UpdateAll s studies
-            //        |> API.Investigation.setStudies investigation
-            //    | None ->
-            //        printfn "Study with the identifier %s does not exist in the investigation file" studyIdentifier
-            //        investigation
-            //| None -> 
-            //    if verbosity >= 1 then printfn "The investigation does not contain any studies"  
-            //    investigation
-            //|> Investigation.toFile investigationFilePath
-    
+            let assayFilePath = IsaModelConfiguration.tryGetAssayFilePath assayIdentifier arcConfiguration |> Option.get
+            
+            let doc = Spreadsheet.fromFile assayFilePath true
+            
+            let persons = MetaData.getPersons "Investigation" doc
+
+            API.Person.add persons person
```

Comment:
Check if person already exists. E.g. `arc s person register` fails if person with the same name already exists. This should be handled consistently.